clang/include/clang/DPCT/DPCTOptions.inc

```diff
@@ -718,6 +718,10 @@ DPCT_ENUM_OPTION(
             "logical-group", int(ExperimentalFeatures::Exp_LogicalGroup),
             "Experimental helper function used to logically group work-items.",
             false),
+        DPCT_OPTION_ENUM_VALUE(
+            "root-group", int(ExperimentalFeatures::Exp_RootGroup),
+            "Experimental helper function used to root group work-items.",
+            false),
         DPCT_OPTION_ENUM_VALUE(
             "nd_range_barrier", int(ExperimentalFeatures::Exp_NdRangeBarrier),
```

Comment:
please deprecated the option:  nd_range_barrier

Reply:
Add Deprecated in option message.